opencraft/tests/test_views.py

```diff
@@ -43,25 +43,25 @@ class IndexViewTestCase(WithUserTestCase):
     url = reverse('index')
     login_url = reverse(settings.LOGIN_URL)
     instance_url = reverse('instance:index')
-    register_url = reverse('registration:register')
+    register_url = login_url
     admin_url = reverse('admin:index')
     admin_login_url = '{}?next={}'.format(reverse('admin:login'), admin_url)
 
     def test_index_unauthenticated(self):
```

Comment:
Unauthenticated users should see the login form (otherwise it gets hard for us to log in the management console).
Users that don't have `manage` permissions should be redirected to the frontend.